charts/ceph-csi-cephfs/README.md

```diff
@@ -71,3 +71,70 @@ If you want to delete the namespace, use this command
 kubectl delete namespace ceph-csi-cephfs
 ```
+
+### Configuration
+
+The following table lists the configurable parameters of the ceph-csi-cephfs
+charts and their default values.
+
+| Parameter | Description | Default |
+| ---------------------- | ---------------- | ---------------- |
+| `rbac.create` | Specifies whether RBAC resources should be created | `true` |
+| `serviceAccounts.nodeplugin.create` | Specifies whether a ServiceAccount should be created | `true` |
+| `serviceAccounts.nodeplugin.name` | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname | "" |
+| `serviceAccounts.provisioner.create` |  Specifies whether a ServiceAccount should be created | `true` |
+| `serviceAccounts.provisioner.name` | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname | "" |
+| `csiConfig` | Configuration for the CSI to connect to the cluster | [] |
+| `logLevel` | Set logging level for csi containers. Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity. | `5` |
+| `nodeplugin.name` | Specifies the name | `nodeplugin` |
+| `nodeplugin.updateStrategy` | Specifies the update Strategy. If you are using ceph-fuse client set this value to OnDelete | `RollingUpdate` |
+| `nodeplugin.priorityClassName` |  set user created priorityclassName for csi plugin pods. default is system-node-critical which is highest priority | `system-node-critical` |
+| `nodeplugin.profiling.enabled` | Specifies whether profiling should be enabled | `false` |
+| `nodeplugin.registrar.image.repository` | Image | `k8s.gcr.io/sig-storage/csi-node-driver-registrar` |
+| `nodeplugin.registrar.image.tag` | Image tag | `v2.2.0` |
+| `nodeplugin.registrar.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+| `nodeplugin.plugin.image.repository` | Image | `quay.io/cephcsi/cephcsi` |
+| `nodeplugin.plugin.image.tag` | Image tag | `canary` |
+| `nodeplugin.plugin.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+| `nodeplugin.nodeSelector` | Kubernetes `nodeSelector` to add to the Deployment | `{}` |
+| `nodeplugin.tolerations` | List of Kubernetes `tolerations` to add to the Deployment | `{}` |
+| `nodeplugin.forcecephkernelclient` | Set to true to enable Ceph Kernel clients on kernel < 4.17 which support quotas | `true` |
+| `nodeplugin.podSecurityPolicy.enabled` | If true, create & use Pod Security [Policy resources](https://kubernetes.io/docs/concepts/policy/pod-security-policy/). | `false` |
+| `provisioner.name` | Specifies the name of provisioner | `provisioner` |
+| `provisioner.replicaCount` | Specifies the replicaCount | `3` |
+| `provisioner.timeout` | Timeout for waiting for creation or deletion of a volume | `60s` |
+| `provisioner.priorityClassName` | set user created priorityclassName for csi provisioner pods. default is system-cluster-critical which is less priority than system-node-critical | `system-cluster-critical` |
```

Comment:
```suggestion
| `provisioner.priorityClassName` | Set user created priorityclassName for csi provisioner pods. default is system-cluster-critical which is less priority than system-node-critical | `system-cluster-critical` |
```

Reply:
Follow a similar format throughout the Description column. It looks much clear ;)